LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -1982,3 +1982,5 @@ imported/w3c/web-platform-tests/webrtc/simulcast/setParameters-active.https.html
 imported/w3c/web-platform-tests/webrtc/simulcast/h264.https.html [ Pass Failure ]
 
 webkit.org/b/268423 [ Sonoma+ Debug arm64 ] http/tests/inspector/dom/didFireEvent.html [ Skip ]
+
+webkit.org/b/268494 [ Monterey+ Release ] media/track/media-element-enqueue-event-crash.html [ Pass Crash ]
```

Comment:
We don't test on anything older than Monterey, so perhaps that part of the annotation can be omitted?

Reply:
I'll submit the change

Reply:
We were actually having a conversation about this in the CI meeting. We felt that it was important to add the OS it is occurring on in the bug and in the expectation so it's easier to audit and tell when this started in the future.